problems/removeNumbersAtOddIndices.js

```diff
@@ -2,8 +2,16 @@
 * @param {number[]} arr - The input array
 * @returns {number[]} - An array removing all elements initially appearing at an odd index
 */
-function removeNumbersAtOddIndices() {
-
+const removeNumbersAtOddIndices = arr => {
+    let evenIdxArray = []
+    for (let i = 0; i < arr.length; i +=2) {
+        if(i % 2 === 0) {
```

Comment:
Try using .filter for cleaner code. Remember the callback on array methods can also take the index as a second argument.